rust/agama-network/src/nm/dbus.rs

```diff
@@ -260,11 +260,11 @@ pub fn merge_dbus_connections<'a>(
     Ok(merged)
 }
 
-fn is_bridge(conn: NestedHash) -> bool {
+fn is_bridge_port(conn: NestedHash) -> bool {
     if let Some(connection) = conn.get("connection") {
         if let Some(port_type) = connection.get("port-type") {
-            if port_type.to_string().as_str() == "bridge" {
-                return true;
+            if let Ok(s) = TryInto::<String>::try_into(port_type) {
```

Comment:
```suggestion
            if let Ok(s) = TryInto::<&str>::try_into(port_type) {
```

Feel free to ignore the suggestion as it is a tiny detail.